db/views/analytics_client_geolocations_v01.sql

```diff
@@ -0,0 +1,18 @@
+SELECT
+    id,
+    client_id,
```

Comment:
Noting a weird thing: for imported Warehouse locations, `client_id` points to a destination client. For HMIS-collected geolocations, `client_id` points to a source client. I think the purpose of this is only to bring along HMIS data, so I think its ok for dbt to just join this to source clients (however/wherever that occurs!).